amlnx_adapter/amlnx_adapter/arangoDB_graph_descriptor.yaml

```diff
@@ -0,0 +1,50 @@
+arangodb:
```

Comment:
Why do we need this file, imo we should not commit concrete login information including PW